default/scripting/policies/RACIAL_PURITY.focs.txt

```diff
@@ -9,25 +9,68 @@ Policy
     effectsgroups = [
         [[SPECIES_LIKES_OR_DISLIKES_POLICY_STABILITY_EFFECTS]]
         
+        // This policy disables influence and research foci for non-capital, non-exobot species
+        // See /species/common/focus.macros
+
+        // Creates a special to ensure that deadopting this policy has bad consequencies for a while.
+        EffectsGroup
+            scope = Source
+            activation = Not HasSpecial name = "RACIAL_PURITY_MASTER_SPECIAL"
+            effects = AddSpecial name = "RACIAL_PURITY_SLAVE_SPECIAL"
```

Comment:
you check for the master and add a slave? also not sure what these terms mean - they got me confused in CC as well.

Reply:
That's a bug. I fixed it in local. Does not work anyways.

Reply:
also looking at the code below I guess your issue is that you should add the master special here